lua/mason-registry/drools-lsp/init.lua

```diff
@@ -0,0 +1,71 @@
+--[[
```

Comment:
This documentation is great, but it's not the best place to put it. I see there's a great deal of documentation provided in the lspconfig server config, hopefully I didn't remove anything that isn't there already?

Reply:
@williamboman It looks all good. I'll switch over my local testing nvim init.lua to use the public github location vs. my local development folders now, and re-test. But again, it looks like the changes are fine. As for the removed documentation at the top of the file, the only thing of real loss was the option of simply configuring the cmd to the exec-shell-wrapper that mason generates. That part is only available to mason users. But that's fine too. I'm going to put more documentation right into the README on the drools-lsp github repo. :)

Reply:
> That part is only available to mason users. But that's fine too. I'm going to put more documentation right into the README on the drools-lsp github repo. :)

Ah, we should be able to provide any necessary extra configuration in mason-lspconfig. [This directory](https://github.com/williamboman/mason-lspconfig.nvim/tree/main/lua/mason-lspconfig/server_configurations) contains configuration overrides provided by Mason.

Reply:
Ah, something like [this](https://github.com/williamboman/mason-lspconfig.nvim/blob/main/lua/mason-lspconfig/server_configurations/groovyls/init.lua) seems like it worked. But how does one leverage that mechanism? Or is it just automatic? I'll create a new branch, make the change, test it, and if all good, submit a new PR for that.

Oh, and btw, I re-tested with your changes straight from github main, and it all works still. So your changes were fine. :)

Reply:
Awesome, thanks!! [Yep it's automatic.](https://github.com/williamboman/mason-lspconfig.nvim/blob/43c7f402cb9822b61b5dc05bf0b385928df256a6/lua/mason-lspconfig/lspconfig_hook.lua#L26) I saw you had a question regarding the directory structure and it's the same structure over there - directories instead of files. It's just a personal preference of mine (in the case of mason-lspconfig it does allow for additional, adjacent, modules should they be needed).

The directory name needs to match up with the lspconfig server name.

Reply:
Although oops, I must have missed putting drools-lsp in https://github.com/williamboman/mason.nvim/blob/main/PACKAGES.md, though... But I think you do regular re-generates, yes?